examples/platform/linux/AppMain.cpp

```diff
@@ -492,6 +497,21 @@ int ChipLinuxAppInit(int argc, char * const argv[], OptionSet * customOptions,
     return 0;
 }
 
+namespace {
+static constexpr size_t kFabricId = 1;
+
+ExampleOperationalCredentialsIssuer gOpCredsIssuer(kFabricId);
+PersistentStorage gStorage;
+
+CHIP_ERROR PrepareJointFabricCluster()
```

Comment:
The entire joint fabric feature should be conditional on build,and should be in a different example, not all-clusters-app. Please do like what Fabric Sync did, and make your own special samples for that whole method that doesn't bloat and complexify the already very complex all-clusters-app